src/hooks.ts

```diff
@@ -0,0 +1,3 @@
+import React, { useReducer, useContext, createContext, Dispatch } from 'react';
+
+export type DispatchT = Dispatch<any>;
```

Comment:
혹시 Dispatch<any>를 안쓰시고 이렇게하신 이유가 있을까요??

Reply:
이것도 이제 본준님처럼 action에 타입을 명시해주면 dispatch 도 각각의 액션 타입에 맞게 작성을 해줘야 하는데.. 귀찮아서 <any> 로 놔둔 것입니댜 ㅎ.. 
원래 같으면 
```suggestion
export type DispatchSomeAction_A_T = Dispatch<SomeAction_A>;
export type DispatchSomeAction_B_T = Dispatch<SomeAction_B>;
```

이런 식으로 해줬어야 하는 건데..